### PR TITLE
101 align and style graphs

### PR DESCRIPTION
**Pull request by moritzpflueger**

### CODE REVIEW

**Comment by moritzpflueger** [comment]
tweaked the chart options a little bit.

also I don't know what the Adr Chart represents. Can you tell me the correct Title for chart-adr.html?

**Comment by pinpox** [comment]
> also I don't know what the Adr Chart represents. Can you tell me the correct Title for chart-adr.html?

ADR is a measure often used for a player's performance in CS:GO and means *A*verage *D*amage per *R*ound. The number is usually around 100, with everything lower than that being considered "bad" and everything above being considered "good" since every player has 100 health at the start of each round. 

It is calculated internaly by as `damage dealt`/`rounds played`. So with a ADR > 100 you basically have done more damage than was inflicted to you.

ADR is strongly related to the other two values: Kill-death-ratio (`KD`) and accuracy (shots fired/shots hit). So we should build a intuitive visualisation of these three values.

**Comment by pinpox** [comment]
Just a thought: We could combine the two charts into one, showing the three data series together. The echarts library can display area and line charts together, so maybe ADR shown as area with the two other values overlayed (or whatever combination of area and line you like)

File: templates/chart-adr.html [hunk 1]
```diff
@@ -7,10 +11,14 @@
 optionAdr = null;
 
 optionAdr = {
-responsive: true,
-maintainAspectRatio: false,
-height: '100%',
-width: '100%',
+    title: {
+        text: 'please update title text if you know what this data represents'
```
Comment (pinpox) [inline templates/chart-adr.html:15]: The title should be something like "Average Damage per Round". You can also leave out the titles for both charts if you combine them and use the legend instead.

File: templates/chart-accuracy.html [hunk 2]
```diff
@@ -16,10 +20,9 @@
         data: ['KD', 'HitRatio']
```
Comment (pinpox) [inline templates/chart-accuracy.html:20]: Here is an example of how to use the legend.

**Comment by moritzpflueger** [comment]
I changed the title for now. I'll open another issue for finding a nice way to merge the two charts or enhance visualization in some other way

**Comment by pinpox** [comment]
Great work, thank you!

**Pull request merged**